line-sdk/build.gradle

```diff
@@ -3,6 +3,8 @@ apply plugin: 'com.novoda.bintray-release'
 apply plugin: 'kotlin-android'
 apply plugin: 'kotlin-android-extensions'
 apply plugin: 'kotlin-kapt'
+apply plugin: 'com.github.dcendents.android-maven'
+apply plugin: 'com.jfrog.bintray'
```

Comment:
trivial:
could you please sort these plugins in alpha-beta order?
looks like all the other plugins sorted in this way